fe/fe-core/src/main/java/org/apache/doris/blockrule/SqlBlockRule.java

```diff
@@ -33,12 +34,13 @@
 import java.util.List;
 import java.util.regex.Pattern;
 
+/**
+ * Use for block some sql by rule.
+ **/
 public class SqlBlockRule implements Writable {
 
     public static final String NAME_TYPE = "SQL BLOCK RULE NAME";
 
-    public static final String DEFAULT_USER = "default";
```

Comment:
why delete this line?

Reply:
DEFAULT_USER is used only when the first designed, and then defined with global, it is no longer needed